### ISSUE TITLE
Add bounce with momentum scroll to top?

### ISSUE THREAD

**Issue by wingedpig** [BoxFactura/pulltorefresh.js#98]
On iOS, when momentum scrolling a div up, it 'hard stops' when it reaches the top. You can then pull down to refresh. Is there any way to add a bit of a bounce, where if you momentum scroll up to the top, it will continue to scroll past the top, starting the process of pulling to refresh, but not completing it, and then bouncing back to the top?

**Comment by faelsoto** [comment]
Uhm, could you record the describe behavior? It's hard to follow just by your description.

Also, this might be an impossible task, since Safari won't report when the user has overshot the viewport when scrolling, it only says 0 (and I think it's great, since having negative values would break lots of things).

**Comment by wingedpig** [comment]
Sorry, I tried to explain best I could. Go to Twitter on your iOS device, using the browser (I assume it works the same on Android, but I don't have one to test). When viewing your timeline, scroll down a little bit, then flick down to scroll back to the top. Notice how when it scrolls all the way to the top of the page, it continues a little bit before bouncing back. This is standard behavior for web pages on mobile devices. And it works on Twitter even with their pull to refresh. When I activate pull to refresh on a div on a web page, scroll a bit and flick down to scroll up, it stops hard at the top, and doesn't 'bounce'.

**Comment by pateketrueke** [comment]
@wingedpig 

I'm not iOS user but I believe the behavior you describe is kinda native for the webview and it cannot be emulated on other devices just with pure Javascript, please let me know if I am wrong.

**Comment by wingedpig** [comment]
Twitter's an existence proof that it can be done. I just don't know how.

**Comment by faelsoto** [comment]
> Sorry, I tried to explain best I could. Go to Twitter on your iOS device, using the browser (I assume it works the same on Android, but I don't have one to test). When viewing your timeline, scroll down a little bit, then flick down to scroll back to the top. Notice how when it scrolls all the way to the top of the page, it continues a little bit before bouncing back. This is standard behavior for web pages on mobile devices. And it works on Twitter even with their pull to refresh. When I activate pull to refresh on a div on a web page, scroll a bit and flick down to scroll up, it stops hard at the top, and doesn't 'bounce'.

Your explanation is on-point, but remember than an image speaks a thousand words, so a video would be great.

> Twitter's an existence proof that it can be done. I just don't know how.

If I were you, I'd start by debugging the code.

I'm closing this because it got sidetracked offtopic, and it doesn't have anything to do with the library. If you find something, feel free to create a PR.
